app/javascript/designSystem/__stories__/avatarsAndLogos.stories.jsx

```diff
@@ -3,9 +3,10 @@ import { storiesOf } from '@storybook/react';
 
 import './typography.scss';
 
-storiesOf('Base/Components', module).add('Avatars & Logos', () => (
-  <div className="container">
-    <div className="body">
+storiesOf('Base/Components/HTML/Avatars & Logos', module).add(
+  'Description',
+  () => (
+    <div className="container">
```

Comment:
So the preview of each variation is not wrapped with `.container` class anymore which is absolutely fine. But in my crayons.html.erb file I had following code:

```css
.container *,
.container *::before,
.container *::after {
  box-sizing: border-box;
}
```
Obviously it does affect how components are rendered. I can see two workarounds:
1. add `box-sizing: border-box;` to every element in every component and ONCE we have it globally, we will just remove it...
2. add it back to some parent container... that one isn't ideal, because in order to use these components we will have to make sure their parent element on the page also has this styling.

I'm leaning towards first solution but maybe do it with some sort of a mixin so it's then easier to track and replace...? 

```scss
@mixin box-sizing($value: border-box) {
  &,
  &::before,
  &::after,
  & *,
  & *::before,
  & *::after {
    box-sizing: $value;
  }
}
```

and then 

```scss
.crayons-[component] {
  @include box-sizing;
  ...
}
```

Dirty but works... Thoughts?

Reply:
I can add it to Storybook CSS file I have and then all good.

Reply:
but it still has to work within DEV codebase.  should I add that mixin to crayons?

Reply:
See https://github.com/thepracticaldev/dev.to/pull/6578/files#diff-b38ebc67ab6952125ae780320692ce41R1

Reply:
Would all our components always be wrapped in a `.container` class? If so I can add a decorator to Storybook for this. It essentially wraps each componentin a div with the container class. It's just a way to avoid doing it multiple times.

Reply:
No, I don't think they will always be wrapped. Components should not be dependent on parent elements. This is exception (workaround) because we can not enable `box-sizing: border-box;` globally yet.

Reply:
Actually you can just do this to only target our components. Once we enable border-box globally, we can just remove it.

```css
[class^='crayons'],
[class^='crayons']::before,
[class^='crayons']::after,
[class^='crayons'] *,
[class^='crayons'] *::before,
[class^='crayons'] *::after {
	box-sizing: border-box;
}
```

Reply:
ohh this is clever! i love it! :) 

i'll add it in a separate PR

Reply:
I've got it in here for now https://github.com/thepracticaldev/dev.to/pull/6578/files#diff-4f059954c7f36561259f093741bd2b66R1, I can PR it up now since I'm currently in it.